### ISSUE TITLE
USA TV

### ISSUE THREAD

**Issue by DryKillLogic** [danamag/stremio-addons-list#409]
### Addon Manifest URL

https://848b3516657c-usatv.baby-beamup.club/manifest.json

### Addon Description

Local channels, news, sports, entertainment, lifestyle, kids, Spanish-spoken, documentaries, and much more.

![preview](https://github.com/user-attachments/assets/375dd289-a080-41a7-ac7c-fc4c5f580459)

### Language of Content

English (en-US)

### Choose Labels

- [ ] anime
- [ ] asian drama
- [ ] bollywood
- [ ] cartoons
- [ ] debrid support
- [ ] http streams
- [X] live tv
- [X] metadata
- [ ] misc
- [ ] movies
- [ ] music
- [ ] nsfw
- [ ] podcasts
- [ ] radios
- [ ] subtitles
- [ ] torrents
- [ ] tv shows
- [ ] video games

**Comment by cletusvandamnit** [comment]
Awesome addon that just works. No configuration needed at all. I just installed and started watching. Installed on Stremio for Nvidia Shield, and have not encountered any issues. Dev was also kind enough to add additional channels that were requested on their Reddit post. Very pleased with this one!

**Comment by DryKillLogic** [comment]
For more information or to get help please check [this post](https://www.reddit.com/r/StremioAddons/comments/1e9vbvt/addon_watch_usa_live_tv/)

**Comment by JJWilson-75** [comment]
Fantastic man. This is the most reliable live TV addon on Stremio.

**Comment by FunkyBoeta** [comment]
Tested on my Samsung phone, MiBox android TV box and on a Samsung smart tv using the native apps for each platform, works flawlessly on all three devices. Good variety of channels on offer, with a more than decent helping of sports channels available. I didn't have much time to test most of the channels for any extended period of time yet, but the ones I tested seems to be performing quite well on my 40mb line. 

Big shoutout to the developer who's been very active on the Stremio subreddit, listening to feedback and adding improvents where possible. Your efforts are greatly appreciated, thank you!

**Comment by 11lee10** [comment]
Awesome add on for Stremio making Streamio more of an all in one app for watching all content. This was a must needed addon and the dev here delivered amazingly! 
Works great on my laptop, smasung tv and iphone with outplayer!

**Comment by markusqw1** [comment]
I've been using this addon for a while now, and I must say, it's fantastic! It allows me to watch all my favorite channels from the US seamlessly. The streaming quality is Great and reliable. Highly recommend this addon to anyone looking for a great addon to watch US TV channels. Great job to the developers and thank you from Australia.

**Comment by Frisco89** [comment]
Wow seems to be actively adding more channels even. Awesome work!

**Comment by darkmanax** [comment]
Just installed this, it works flawlessly! So glad to finally have some live tv on Stremio! Thanks for all your work into this. Hopefully this is very well received and continues to support existing and more content.

**Comment by DrekAnon** [comment]
This is really great, it's missing MotorTrend though and would be perfect if you can add it! Thanks.

**Comment by exeramosm** [comment]
Amazing job! thanks!!!

**Comment by TheGlitch1** [comment]
I don't know what to say, 

I woke up this morning just wanted to work while watching tv in one place ... kept reading Reddit posts until I found your comment suggesting to check your addon ... oh god , you just made my day.

**Comment by dieselshroom** [comment]
Fantastic! Thank you for this. 🙏

**Comment by ksbox** [comment]
thank you so much!

**Comment by shugar414** [comment]
So appreciate the loads of time and effort that you put into creating this amazing Stremio add-on. Your curated list of live TV is brilliant!

Thank you!

**Comment by juliendogny** [comment]
Hello,
Would it be possible to update abc 9 Orlando's link, it does not seem to work ?

**Comment by boxheadmoose** [comment]
The best add-on!  Thank you for the time you've devoted to this project and for all your hard work!

**Comment by chasedolla** [comment]
The author of this addon is probably the nicest guy I’ve come across on Reddit. This addon is top tier and has been a game changer. Hopefully it stays active where there isn’t enough traffic to start causing issues but so far so good and he’s even gone out his way to bless people with requests he did not have to take

**Comment by joepnxsun** [comment]
Hello, Is it possible to add F1 and MotoGP streams on this?

**Comment by 100banana** [comment]
Created an account just to say thank you. You are the best!

**Comment by Damejames65** [comment]
Dude, this is fucking amazing

**Comment by darkpnx00** [comment]
Thank you so much for this! greatest add-on for live tv. hopefully someone will add tnt sports on this. thanks again for this wonderful add-on

**Comment by thelastblt** [comment]
I just wanted to again say thank you for this add on. Seriously such a great add on. 

I can tell a lot of effort went into this and it shows. Thank you for all the work to make this amazing.

**Comment by rece600** [comment]
Thank you so much for the effort you put into this add-on!

**Comment by TuupacFR** [comment]
It's simply too good to stay but I hope it does !!

**Comment by debonair99** [comment]
Awesome job with this addon! Is there any way to get rid of the captions? They sometimes get in the way when watching sports

**Comment by DoctahBuddah** [comment]
best addon there is no doubt. had to create an account just to post this. the dev has added a bunch of channels & theyre all stable. thank you dev!

**Comment by ksw22** [comment]
Fantastic dev! So grateful for all the time and effort that you put into this add-on. You're the GOAT 🙌🏽

**Comment by Schlog-Guy** [comment]
This is such a cool tool! What an incredible gift to the community. Hats off to you for the immense amount of work that went into this project. It's truly a game changer!

**Comment by debonair99** [comment]
> Awesome job with this addon! Is there any way to get rid of the captions? They sometimes get in the way when watching sports

Is it feasiible to remove the closed captions??

**Comment by luniz7** [comment]
One of the best add ons out there! Great job!!

**Comment by jbrantley130** [comment]
Awesome add-on. By far the best add-on for live TV. I love that the dev has added channels after the initial launch. I can't really think of any channels that me and my family like that aren't already on USA TV. 

Dev is a super nice guy too. I've asked him if he could add 3 channels and he has added every single one of them. 

The add-on also just simply works with no setup needed. Just install and watch all of the cable channels for free. Amazing work by DryKil1Logic.

**Comment by jbrantley130** [comment]
> > Awesome job with this addon! Is there any way to get rid of the captions? They sometimes get in the way when watching sports
> 
> Is it feasible to remove the closed captions or are they a permanent feature?

I don't have any problems with closed captions on USA TV. I'm using the $20 onn 4k streaming box.

**Comment by jbrantley130** [comment]
> best addon there is no doubt. had to create an account just to post this. the dev has added a bunch of channels & theyre all stable. thank you dev!

Yes and they're all in 1080p HD it looks like to me. I know all of the ESPN network channels are. I get to watch all of my college football games in HD. SportsFire which is another app, not even a Stremio add-on, has a lot of sports channels but the picture quality is crap compared to USA TV.

**Comment by cdkao** [comment]
AMAZING ADDON!  Thank you so much!

**Comment by jbrantley130** [comment]
> AMAZING ADDON! Thank you so much!

I think you meant to reply to the OP. I'm def not the creator of USA TV. That would be DryKil1Logic and he's an awesome dude.

**Comment by cdkao** [comment]
> > AMAZING ADDON! Thank you so much!
> 
> I think you meant to reply to the OP. I'm def not the creator of USA TV. That would be DryKil1Logic and he's an awesome dude.

Aww yes.. thank you for the correction!  

Thank you DryKillLogic!  Well done!

**Comment by debonair99** [comment]
Anyone have any suggestions how to get rid of the closed captions? They are always there when on my Samsung TV but not present when using a PC? Any help appreciated. Cheers

https://github.com/user-attachments/assets/7bb484fc-8309-4623-92e5-55facb6402a1

**Comment by Germsicle** [comment]
> Anyone have any suggestions how to get rid of the closed captions? They are always there when on my Samsung TV but not present when using a PC? Any help appreciated. Cheers
> 
>  IMG_6781.1.mov

Sounds like it's your tv settings, not the app.

**Comment by autumnalpilot** [comment]
Hi, and thank you for one of the best addons for stremio! I have a request and two questions.
Request:
1. Can you please add The CW channel, since the existing one is Pix11 which is not the real proper The CW? Basically the real one is this: [The CW](https://dlhd.so/stream/stream-300.php)

Question:
1. All your channels of your addon are just free tvs or pay tvs too? Because I don't want to occur into risks. If any is not free, can you please provide a list of free and pay ones, in order to let, who wants to avoid problems, to know which to open and which not?
2. How many seconds is the delay of channels from real cable or satellite tv? (For example Disney Channel or Nickelodeon or Boomerang)

Thank you very very much!

**Comment by diznanl** [comment]
Excellent work ... thumbs up here for the stremio addons page. Keep up the great work!

**Comment by spyrosjudas** [comment]
Absolutely insane add-on!!!
All the hard for this quality project is evident and I appreciate the fact that you're actively maintaining and updating this!

Keep it up!

**Comment by cheebee333** [comment]
Good quality live tv add-on. You have a nice very good list of channels here. Thank you for your work and continuing supporting the app. Cheers

**Comment by FLY0830** [comment]
I am getting a playback error on the US Channels with more than one selection, i.e. CBS, ABC, FOX.  Any recommendation on how to fix this?

**Comment by dustiercookie** [comment]
> I am getting a playback error on the US Channels with more than one selection, i.e. CBS, ABC, FOX. Any recommendation on how to fix this?

Having a similar issue, but it is localized to CBS streams only. Anyone else experiencing this?

**Comment by bjclopes** [comment]
This add-on is so cool, I really appreciate all of the work you've but into it!
I've read through your Reddit post and understood that you got a lot of overwhelming requests, so I'd like to avoid that. This looks already a whole lot to take on already.

I was wondering if you'd ever consider sharing the source code so others could try and implement a similar add-on with streams from their respective countries of interest. If I was ever successful and wanted to publish such an add-on, you would, of course, be credited as the mais contributor.

Many thanks for your hard work either way! :)

**Comment by Synapse-web** [comment]
One of the best add-ons period! Thank you so much!!!  Do you Patreon? I saw you are only maintaining the project now but was wondering not for you to implement but to know if it is possible to add the program info and overlay on the channel logo like the RPDB rating add-on. USA live TV would be the IPTV killer. Thanks again for sharing your great work!

**Comment by TuupacFR** [comment]
> Hello,
> 
> You think its possible to add other language TV like French ?

Yes please that would be amazing if you can add French channels 🙏🏼

**Comment by tylertylerday** [comment]
Looks like it was taken down or not working now?

**Comment by diznanl** [comment]
Not complaining -- just an FYI:

ESPN HD server 1: Has no sound
ESPN HD server 2: Usually has another channel on
ESPN HD server 3: Works great!
ESPN HD server 4: Has a Russian page saying, ""To resume viewing, you need to pay for a subscription."

**Comment by Frisco89** [comment]
Seems to be broken.

**Comment by DrekAnon** [comment]
Can confirm, seems currently broken

**Comment by tjand44** [comment]
Great stuff. Tried about 10 channels. All work great except Fox News. Is that one just temporarily down?

**Comment by dotkeenan** [comment]
Really good.  But lately NBC Sports doesn't work.

**Comment by geneh56** [comment]
NBC Sports doesn't work anywhere near as well as it did prior to v1.4. Various errors but some include a message about unsupported video format - and that's on FireTV and Windows PC. Hope you discovery why. Thanks for all you do!

**Comment by JohanLibert-ctrl** [comment]
Thank you for everything

**Comment by Alexislml96** [comment]
Working great.

Only seems to have 1 issue with TUDN stream, audio is not synced with image.

Not sure if it’s the video source

**Comment by Soapm113** [comment]
This is an absolute must have addon, this makes IP TV almost like cable TV, except without the bill.

**Comment by Alexislml96** [comment]
Hi devs thanks for a great add on.

Do you have plans to fix/add again the TUDN channel?

I’m Mexican so that channel is the one I watch for live sports. 

Thanks in advance.

**Comment by xetrics** [comment]
Nice work.

**Comment by vvrogers4** [comment]
This is amazing! I only seem to be able to find ~90 channels on stremio. Missing NESN, which would be a big deal for my NE wife. Is there a trick to get all the channels or something I'm missing? Already liked the add on and will be sharing with all my friends!
